sdks/java/io/snowflake/src/main/java/org/apache/beam/sdk/io/snowflake/SnowflakeIO.java

```diff
@@ -169,7 +182,18 @@
   private static final Logger LOG = LoggerFactory.getLogger(SnowflakeIO.class);
 
   private static final String CSV_QUOTE_CHAR = "'";
-  private static final String WRITE_TMP_PATH = "data";
+
+  static final int DEFAULT_FLUSH_ROW_LIMIT = 10000;
+  static final int DEFAULT_STREAMING_SHARDS_NUMBER = 1;
+  static final int DEFAULT_BATCH_SHARDS_NUMBER = 0;
+  static final Duration DEFAULT_FLUSH_TIME_LIMIT = Duration.millis(30000); // 30 seconds
+  static final long SLEEP_STREAMING_LOGS_IN_SEC = 5000;
+  static final Duration STREAMING_LOGS_MAX_SLEEP = Duration.standardMinutes(2);;
+
```

Comment:
Maybe is worth to make those constant names more consist ? 
DEFAULT_**STREAMING**_FLUSH_ROW_LIMIT
DEFAULT_STREAMING_SHARDS_NUMBER
DEFAULT_BATCH_SHARDS_NUMBER
DEFAULT_**STREAMING_**FLUSH_TIME_LIMIT
STREAMING_LOGS_SLEEP
STREAMING_LOGS_MAX_SLEEP

+
why STREAMING_LOGS_SLEEP is not using Duration?

Reply:
Sure, I will change.